### PR TITLE
Revert change on NEP-1073 cause table to repeat content after switching pages

### PR DESCRIPTION
**Pull request by rauxalach**
screen shot 2017-11-14 at 9 43 17 am

### CODE REVIEW

**Comment by grantspeelman** [comment]
@rauxalach just straight reverting will return the issue of [NEP-1073](https://jobready.atlassian.net/browse/NEP-1073) #13
Should we not be looking at doing something that keeps NEP-1073 fixed?

**Comment by rauxalach** [comment]
@grantspeelman Talked to @tomt-jobready , he suggests to just bump this into new version so Neptune will keep using the NEP-1073 #13 version that they think their issue is fixed. 

I am not too sure about how Neptune is using osom-table but their osom-table version locked at 3.1.4

Direct using osom-table at version 3.1.2 so I guess Direct can have same issue as I mention if they bump to newest version !!!

File: app/assets/javascripts/osom-tables.js [hunk 1]
```diff
@@ -150,8 +150,8 @@
 
   // Callback for successful ajax load of table data
   function table_load_success(container, new_content, url) {
-    var new_container = $('<div class="osom-table ">').append($(new_content).filter('div.osom-table'));
-    container.html(new_container);
+    var new_container = $(new_content);
+    container.html($(new_content).children());
```
Comment (dangquangthai) [inline app/assets/javascripts/osom-tables.js]: just 1 line `var new_container = container.replaceWith(new_content);`

Comment (rauxalach) [reply app/assets/javascripts/osom-tables.js]: @dangquangthai I think `replaceWith` produce different result to `html`. In this PR , I intend to revert to what stable before the #13 , any refactor or code optimize is welcome in another PR, IMHO. 👍

**Pull request merged**